File: 11653.py
import sys
sys.stdin = open("input.txt", 'r')

n = int(input())

i = 2
while i**2<=n:
    if n%i==0:
        n //= i
        print(i)
    else:
        i += 1

if n>1:
    print(n)

File: _11000/11653.cpp
#include <iostream>

using namespace std;

int main() {
    cin.tie(NULL);
    ios_base::sync_with_stdio(0);
    freopen("input.txt", "r", stdin);

    int n;
    cin >> n;

    for(int i=2; i*i<=n; i++) {
        while(n%i == 0) {
            cout << i << '\n';
            n /= i;
        }
    }

    if(n > 1) {
        cout << n << '\n';
    }

    return 0;
}